From 334550f49998ec53212daf151eb4e0829438cd16 Mon Sep 17 00:00:00 2001
From: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Date: Wed, 21 Apr 2021 13:24:07 +0200
Subject: [PATCH 0257/1141] arm64: defconfig: Enable stm32_rproc support

This patch adds in built-in the remoteproc driver used by STM32MP SOCs.

Signed-off-by: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Change-Id: Ied8b64a5866e120452371fc3178626136a6a9290
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/310426
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Reviewed-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Domain-Review: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -53,6 +53,7 @@ CONFIG_STM32_MDF_ADC=m
 CONFIG_STM32_MDF_CORE=m
 CONFIG_STM32_OMI=y
 CONFIG_STM32_OMM=y
+CONFIG_STM32_RPROC=y
 CONFIG_STM32_TIMER_CNT=m
 CONFIG_STMMAC_ETH=y
 CONFIG_STMMAC_PLATFORM=y
